src/igvfd/searches/configs/whole_organism.py

```diff
@@ -51,6 +51,7 @@ def whole_organism():
                     'biomarkers.classification',
                     'biosample_term.term_name',
                     'disease_terms.term_name',
+                    'virtual',
```

Comment:
L29, can you updated title to "Disease Terms" for consistency?

Reply:
L100, please remove submitted_by

Reply:
fixed